mitodl/course-search-utils
# Replace icons with Remixicon SVGs

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/5472

### Description (What does it do?)


Replaces [Material UI font](https://fonts.googleapis.com/icon?family=Material+Icons) icons with Remixicon SVG icons installed directly to this package, see @ChristopherChudzicki 's recommendation: (option 2) https://github.com/mitodl/hq/issues/5472#issuecomment-2364030819.

This allows us to remove the font as a peer dependency in projects. In the case of MIT Learn is is imported only for use by this package.

### Screenshots (if appropriate):



Chevrons:
<br />
<img width="337" alt="image" src="https://github.com/user-attachments/assets/708d42b7-6e8b-44a4-be81-9b2cdd73a011">
<br />
<img width="334" alt="image" src="https://github.com/user-attachments/assets/b31b177f-0114-4128-bc18-2841c7196b62">
<br />
<img width="334" alt="image" src="https://github.com/user-attachments/assets/b4497a84-072a-49cb-ab04-19cff02ecbf9">
<br />
<img width="338" alt="image" src="https://github.com/user-attachments/assets/c7aa99b5-6d47-4367-bcdf-36050cfc0317">
<br />
Close button:
<br />
<img width="335" alt="image" src="https://github.com/user-attachments/assets/e0e05ed7-fe5b-4377-8ad4-f06b885b53a9">
<br />
<img width="332" alt="image" src="https://github.com/user-attachments/assets/22acbc57-16c7-4950-a9c3-bdcb0d16d3af">



### How can this be tested?


The accompanying PR on MIT Learn: https://github.com/mitodl/mit-open/pull/1617 install from this PR branch for testing.

Icons should appear as in screenshots above.

### Additional Context
